introduction.md

```diff
@@ -1,3 +1,9 @@
 Oh Hai. \:D/ 
 
 Favorite Code School path: Ruby
+
+Deadly Skills:
+* PHP
+* Ruby
+* JavaScript
+* Google
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?